packages/chain-adapters/src/index.ts

```diff
@@ -1,3 +1,5 @@
 export * from './ChainAdapterManager'
 export * from './api'
 export * from './utils'
+export * as cosmossdk from './cosmossdk'
+export * as ethereum from './ethereum'
```

Comment:
Might as well add a bitcoin export here as well in case we need it in the future.